tests/unit-tests/test_gnn.py

```diff
@@ -580,6 +581,7 @@ def test_gat_node_prediction():
                                      train_ntypes=['_N'], label_field='label',
```

Comment:
Should we change the "_N" here to "DEFAULT_NTYPE"?

Reply:
Let me create another PR to update all the places.